cmd/api-errors.go

```diff
@@ -800,6 +801,12 @@ var errorCodes = errorCodeMap{
 		Description:    "x-amz-object-lock-retain-until-date and x-amz-object-lock-mode must both be supplied",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrOBDLockTimeout: {
+		Code:           "UnsupportedOperation",
```

Comment:
Custom errors in admin API should be prefixed with `XMinioAdmin` to avoid conflicting with S3 error codes.

Reply:
Gotcha